src/app/move-popup/press-tip.directive.js

```diff
@@ -30,7 +30,7 @@ export function PressTip() {
       $element.on('mouseenter', (e) => {
         timer = setTimeout(() => {
           showTip();
-        }, 300);
+        }, 100);
```

Comment:
considered removing this timeout

Reply:
I like having a bit of a pause so stuff isn't jumping out at me. What did you think when you tried it?

Reply:
I do like the instant feedback, but I could go either way. If it was a hover on items then it would be more annoying, but the intent is usually something that I want to see, I think

Reply:
That's fair.